src/webportal/src/app/job-submission/job-submission-page.jsx

```diff
@@ -356,6 +380,29 @@ export const JobSubmissionPage = ({
       .catch(alert);
   }, [jobInformation.virtualCluster]);
 
+  const onTemplateChange = useCallback((_, item) => {
+    if (item.key === 'No') {
+      return;
+    }
+    const jobConfig = JobProtocol.fromYaml(item.protocol);
+    const [
+      jobInfo,
+      taskRoles,
+      parameters,
+      ,
```

Comment:
This comma seems redundant.

Reply:
It is used to ignore some returned values: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment